src/routers/UpdateUserRouter.ts

```diff
@@ -56,6 +56,21 @@ class UpdateUserRouter extends AuthenticatedAppplicationRouter<void> {
       if (!major) throw Error('CornellMajor with that name not found');
       userFields.major = major;
     }
+
+    if (userFieldKeys.length < 1) {
+      throw Error('At least one user field is required');
+    }
+
+    for (const key of userFieldKeys) {
```

Comment:
probably better to use a `map`

Reply:
i actually don't think this bit of code is even necessary; going to redact.